### PR TITLE
Create FreshPorts.adoc

### PR DESCRIPTION
**Pull request by dlangille**

### CODE REVIEW

File: 2021q2/FreshPorts.adoc [hunk 1]
```diff
@@ -0,0 +1,59 @@
+== Status Report Sample - This Will Become The Title
```
Comment (pauamma) [inline 2021q2/FreshPorts.adoc:1]: Put an actual title there?

File: 2021q2/FreshPorts.adoc [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+Link: link:http://www.example.com/project/url[Link description here]
```
Comment (pauamma) [inline 2021q2/FreshPorts.adoc:3]: Likewise, this should go.

File: 2021q2/FreshPorts.adoc [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+### Help wanted ###
+
+Since the link:last report[https://www.freebsd.org/status/report-2020-10-2020-12.html#FreshPorts],
+link:Jethro Nederhof[https://github.com/jethron] has been 
+doing fantastic work bringing the website up to HTML5 and into the modern world.
+Nothing dramatic, from what the users see, as it has been mostly behind-the-scenes.
+
+We can always use more helpers. The website mostly runs itself and requires very little
+on-going maintence (pkg upgrades, OS patche, etc). Most of the work is designing new
```
Comment (pauamma) [inline 2021q2/FreshPorts.adoc:39]: maintenance

File: 2021q2/FreshPorts.adoc [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+Tasks include:
+
+* There is a long list of link:https://github.com/FreshPorts/freshports/issues[issues]
+for the website.
+
+* The link:https://github.com/FreshPorts/git_proc_commit/issues[git_proc_commit project]
+also has a set of issues, mostly in Pyton, some perhaps related to the website.
+
+* Documentation outlining how the various project fit together and how they work is
```
Comment (pauamma) [inline 2021q2/FreshPorts.adoc:51]: projects (number agreement)

**Comment by dlangille** [comment]
Thank you @pauamma for those fixes.

**Comment by dlangille** [comment]
@pauamma how can I properly review the rendered .adoc?

**Comment by pauamma** [comment]
> @pauamma how can I properly review the rendered .adoc?

I don't know, but I passed on the question.

**Comment by bsdimp** [comment]
Can't you just open a .adoc file on github?

**Comment by dlangille** [comment]
I tried that. It doesn't [look right to me](https://github.com/freebsd/freebsd-quarterly/pull/371/files/82fce6f28d60af85e62d58e280482edbfaef8f5c?short_path=078dc97#diff-078dc97c5a73cc52a6d297e20e19ce59c3a48e58f07628bc4822907d7bde7387)

**Comment by bsdimp** [comment]
Maybe try asciidoclive.com

**Comment by dlangille** [comment]
It does not convert the links.

File: 2021q2/FreshPorts.adoc [hunk 5]
```diff
@@ -0,0 +1,57 @@
+== FreshPorts
+
+Link:	link:https://freshports.org/[FreshPorts]
+Link:	link:https://news.freshports.org/[FreshPorts blog]
```
Comment (bsdimp) [inline 2021q2/FreshPorts.adoc:4]: In the asciidoc in the tree, there is no space after the colon

**Comment by dbaio** [comment]
> @pauamma how can I properly review the rendered .adoc?

Hi @dlangille .

For a single .adoc use this to test:

```
pkg install rubygem-asciidoctor
fetch [...FreshPorts.adoc
asciidoctor FreshPorts.adoc && open FreshPorts.html
```

File: 2021q2/FreshPorts.adoc (hunk 5 above)
Comment (ceri) [reply 2021q2/FreshPorts.adoc:4]: I sent you a pull-request with these corrected.

**Comment by dlangille** [comment]
> Can't you just open a .adoc file on github?

It turns out, that if the .adoc file contains proper asciidoc, yes, you can. Thank you.

File: 2021q2/FreshPorts.adoc (hunk 5 above)
Comment (dlangille) [reply 2021q2/FreshPorts.adoc]: Thank you.

**Pull request closed without merging**